src/common/conditional_compilation/include/openvino/cc/pass/itt.hpp

```diff
@@ -21,6 +21,14 @@ OV_CC_DOMAINS(ov_pass);
 #    define MATCHER_SCOPE(region)         const std::string matcher_name(OV_PP_TOSTRING(region))
 #    define RUN_ON_MODEL_SCOPE(region)    OV_SCOPE(ov_pass, OV_PP_CAT(region, _run_on_model))
 
+#    define ADD_MATCHER_SCOPE_FOR_THIS(nspace, region, ...)        add_matcher<nspace::region>(__VA_ARGS__);
+#    define ADD_MATCHER_SCOPE_FOR_THIS_WITHOUT_NSPACE(region, ...) add_matcher<region>(__VA_ARGS__);
+#    define ADD_MATCHER_SCOPE(obj, nspace, region, ...)            obj->add_matcher<nspace::region>(__VA_ARGS__);
+#    define ADD_MATCHER_SCOPE_WITHOUT_NSPACE(obj, region, ...)     obj->add_matcher<region>(__VA_ARGS__);
+#    define REGISTER_PASS_SCOPE(obj, nspace, region, flag, ...)    obj.register_pass<nspace::region>(__VA_ARGS__);
+#    define REGISTER_DISABLED_PASS_SCOPE_WITH(obj, nspace, region, ...) \
+        obj.register_pass<nspace::region, false>(__VA_ARGS__);
+#    define REGISTER_PASS_MODEL_SCOPE_IF(region)
```

Comment:
It is not critical, but I propose to remove `SCOPE` word from the name. It will look more clear.